roles/backup_select_resources/README.md

```diff
@@ -0,0 +1,65 @@
+backup_select_resources
+==================
+
+A role to configure backups for selected resources. The role requires an existing backup vault and plan, and adds selected resources to the provided plan. A set of variables for resource selections is included for use as-is or as examples for modification. This role can be combined with the backup_plan_create role to create or update a backup plan if one does not already exist.
+
```

Comment:
Maybe I missed the reference, but I think you should mention that the user can use the role 'backup_create_plan' together with the 'backup_select_resources' and how the user can do that. I assume that's possible.